src/testInternalBuilds.ts

```diff
@@ -0,0 +1,39 @@
+import * as express from 'express';
+
+import { eb } from './eb';
+import { ExtendedRequest } from './typings/extendedRequest';
+
+const testInternalBuilds = express();
+
+testInternalBuilds.use(async (req, res, next) => {
+  try {
+    const requestedEnvName: string | undefined = req.query.env;
```

Comment:
I think the query param should be called `branch`

Reply:
Renamed